crates/red_knot_python_semantic/src/types.rs

```diff
@@ -424,27 +416,27 @@ impl<'db> Type<'db> {
             (Type::Never, _) => true,
             (_, Type::Never) => false,
             (Type::IntLiteral(_), Type::Instance(class))
-                if class.is_stdlib_symbol(db, "builtins", "int") =>
+                if class.is_known_class(db, KnownClass::Int) =>
```

Comment:
Is there a reason you went for `is_known_class` rather than `is_known` for this method? It's ok either way, just feels like an unnecessary level of repetition of the word "class" when we have `class.is_known_class(db, KnownClass::...)`

Reply:
No reason, is known is good!

Reply:
@carljm should be fixed!